docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -52,17 +52,52 @@ An object that defines any variables your tests require.
 [[synthetics-configuration-playwright-options]]
 = `playwrightOptions`
 
-For available options, see the https://playwright.dev/docs/test-configuration[Playwright documentation].
+For all available options, refer to the https://playwright.dev/docs/test-configuration[Playwright documentation]. 
+
+Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including timeouts, timezones, and device emulation.
+
+[discrete]
+[[synthetics-configuration-playwright-options-timeouts]]
+== Timeouts
 
-[NOTE]
-====
 Playwright has two types of timeouts that are used in Elastic Synthetics:
 https://playwright.dev/docs/test-timeouts#action-and-navigation-timeouts[action and navigation timeouts].
 
 Elastic Synthetics uses a default action and navigation timeout of 50 seconds.
 You can override this default using https://playwright.dev/docs/api/class-testoptions#test-options-action-timeout[`actionTimeout`] and https://playwright.dev/docs/api/class-testoptions#test-options-navigation-timeout[`navigationTimeout`]
 in `playwrightOptions`.
-====
+
+[discrete]
+[[synthetics-configuration-playwright-options-timezones]]
+== Timezones
+
+By default monitors will run in the ??? timezone, but you can specify a timezone in
```

Comment:
It wasn't clear to me when reading the issue what the default behavior is if this is not explicitly set.

Reply:
@colleenmcginnis unfortunately that's part of the problem.  On the service we're not currently setting this, so a default of `Etc/Unknown` is being _used_ which is not valid (see https://github.com/elastic/synthetics-service/issues/1312).

Many sites won't care, but if the site needs to check the timezone, they may fail because of this.

Reply:
Ohh I see.